airflow/providers/jira/provider.yaml

```diff
@@ -36,7 +36,7 @@ versions:
   - 1.0.0
 
 dependencies:
-  - apache-airflow>=2.2.0
+  - apache-airflow>=2.3.0
```

Comment:
I opened a PR to drop this provider

Reply:
I wille get to that PR shortly, but let's get it changed here in the meantime for consistency.